client-data/tools/download/download.js

```diff
@@ -0,0 +1,64 @@
+/**
+ *                        WHITEBOPHIR
+ *********************************************************
+ * @licstart  The following is the entire license notice for the
+ *  JavaScript code in this page.
+ *
+ * Copyright (C) 2020  Ophir LOJKINE
+ *
+ *
+ * The JavaScript code in this page is free software: you can
+ * redistribute it and/or modify it under the terms of the GNU
+ * General Public License (GNU GPL) as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.  The code is distributed WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU GPL for more details.
+ *
+ * As additional permission under GNU GPL version 3 section 7, you
+ * may distribute non-source (e.g., minimized or compacted) forms of
+ * that code without the copy of the GNU GPL normally required by
+ * section 4, provided you include this license notice and a URL
+ * through which recipients can access the Corresponding Source.
+ *
+ * @licend
+ */
+
+(function download() { //Code isolation
+
+    function downloadSVGFile(evt) {
+        var canvasCopy = Tools.svg.cloneNode(true);
+        var styleNode = document.createElement("style");
+
+        // Copy the stylesheets from the whiteboard to the exported SVG
+        stylesheets = document.styleSheets;
+        styleText = "";
+        Array.from(stylesheets).forEach(stylesheet => {
```

Comment:
We support older browsers without array functions

Reply:
What browser versions should we target for compatibility when  writing PRs?

Reply:
wbo works on IE9.
It includes recent js polyfills for new es features: https://github.com/lovasoa/whitebophir/blob/master/server/server.js#L183-L199

But the es syntax itself has to be parsable by IE9

Reply:
done